app/src/main/java/io/pslab/models/PSLabSensor.java

```diff
@@ -92,7 +92,7 @@
     public static final String LUXMETER_DATA_FORMAT = "%.2f";
     public static final String BAROMETER = "Barometer";
     public static final String BAROMETER_CONFIGURATIONS = "Barometer Configurations";
-    public static final String BAROMETER_DATA_FORMAT = "%.5f";
+    public static final String BAROMETER_DATA_FORMAT = "%.2f";
```

Comment:
@Smit2010  please confirm with some mentor regarding the precision required. Should it be .2f or .5f

Reply:
Oh yes @neel1998. @CloudyPadmal could you please suggest the proper precision to be displayed here?
Thanks.